x/oracle/keeper/msg_server.go

```diff
@@ -0,0 +1,169 @@
+package keeper
+
+import (
+	"context"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
+
+	"github.com/umee-network/umee/x/oracle/types"
+)
+
+type msgServer struct {
+	Keeper
```

Comment:
I've noticed that in `x/leverage`, this line appears as `keeper Keeper`,

and lines in other functions would change from `ms.SetFeederDelegation(..)` to `ms.keeper.SetFeederDelegation(..)` for example.

Is exporting this field a meaningful detail from a security perspective @alexanderbez ? (Related to the convention that keeper references are only given to specific modules that need them)

Reply:
Note: it looks like `x/bank` [doesn't care](https://github.com/cosmos/cosmos-sdk/blob/v0.44.0/x/bank/keeper/msg_server.go), so this probably isn't an issue.